MedTracker/Controllers/PatientController.cs

```diff
@@ -117,6 +117,31 @@ public async Task<IActionResult> GetTreatments()
 
     }
 
+    [HttpGet("GetDoctors")]
+    public async Task<IActionResult> GetDoctors()
```

Comment:
If we don't use any input data, at least we should require the user to authenticate itself.